pages/vi/vi-github-issues.md

```diff
@@ -90,7 +90,15 @@ If you're still confused, that's quite alright. [Forking Workflow](https://www.a
 
 ### 5.1. Preview Changes Locally
 
-After making modifications to your local files and before making a commit, you may want to preview the changes locally. To quickly set up a local HTTP server, you can follow [MDwiki's guide here](https://dynalon.github.io/mdwiki/#!./faq.md#Q:_I_don't_want_to_install_nginx_or_apache_locally._What_is_the_fastest/easiest_way_to_setup_a_local_HTTP_server_to_get_started_with_MDwiki_?). However, instead of using the outdated Python method mentioned in the guide, you can use the following command: `python3 -m http.server 8080`.
+After making modifications to your local files and before making a commit, you may want to preview the changes locally. You can also setup a local HTTP Server if you like. To setup a local HTTP Server go to your forked repository, and run the following commands:
+
+If you have Python 3 installed:  
```

Comment:
Hi @sahilvunnam

If you notice the changes on this line, it's an indication that you ran `git checkout -b 3446-movingStep6InstructionsIntoStep6Page` while on `3432-removingHTTPServerLink` instead of `master`. Could you follow the steps on https://open-learning-exchange.github.io/#!./pages/vi/vi-github-issues.md#4._Synchronize_Your_Repository and rework on this change, open a new pr?

Reply:
Yes, I can sorry about that